inc/Calculator.h

```diff
@@ -11,6 +11,21 @@
 #define CALCULATOR_H
 
 #include <iostream>
-// Add your function declarations here.
+namespace Calculator
+{
+    /**
+     * @brief This is the Class containing the functions and variables used in week 1
+     */
+    class Week1
```

Comment:
By convention, class should match file name and be Pascal Case, i.e. `Calculator`